triplets/community/x64_arm64-osx.cmake

```
@@ -0,0 +1,5 @@
+set(VCPKG_TARGET_ARCHITECTURE x86_64 arm64)
```

I think this should be:

```suggestion
set(VCPKG_TARGET_ARCHITECTURE x64 arm64)
```